testing/forge-script/forge-config.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: BUSL-1.1
+#
+# Copyright (C) 2024, Berachain Foundation. All rights reserved.
+# Use of this software is governed by the Business Source License included
+# in the LICENSE file of this repository and at www.mariadb.com/bsl11.
+#
+# ANY USE OF THE LICENSED WORK IN VIOLATION OF THIS LICENSE WILL AUTOMATICALLY
+# TERMINATE YOUR RIGHTS UNDER THIS LICENSE FOR THE CURRENT AND ALL OTHER
+# VERSIONS OF THE LICENSED WORK.
+#
+# THIS LICENSE DOES NOT GRANT YOU ANY RIGHT IN ANY TRADEMARK OR LOGO OF
+# LICENSOR OR ITS AFFILIATES (PROVIDED THAT YOU MAY USE A TRADEMARK OR LOGO OF
+# LICENSOR AS EXPRESSLY REQUIRED BY THIS LICENSE).
+#
+# TO THE EXTENT PERMITTED BY APPLICABLE LAW, THE LICENSED WORK IS PROVIDED ON
+# AN “AS IS” BASIS. LICENSOR HEREBY DISCLAIMS ALL WARRANTIES AND CONDITIONS,
+# EXPRESS OR IMPLIED, INCLUDING (WITHOUT LIMITATION) WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND
+# TITLE.
+
+deployment:
+  repository: "github.com/nidhi-singh02/solidity-scripting/"  # follow the same format as defined here.
+  script_path: "script/NFT.s.sol"  # this must be relative to the repository path
```

Comment:
why not just use the regular DeployAndCallERC20.s.sol already in `/contracts`

Reply:
agreed, will make it work for both local contracts as well as contracts hosted on github. Will test DeployAndCallERC20.s.sol.

Reply:
For local files/folders, there isn't a direct way to access them via kurtosis unless they are part of the package itself. Tested ` DeployAndCallERC20.s.so` through Github url (forked beacon-kit under my profile). It works fine.